### ISSUE TITLE
how to make sampler.set_epoch(epoch) in accelerate

### ISSUE THREAD

**Issue by DogeWatch** [huggingface/accelerate#654]
pytorch ddp make a method to make shuffling work properly across multiple epochs
(https://pytorch.org/docs/stable/data.html#torch.utils.data.distributed.DistributedSampler)
is there a way to do the same thing with accelerate ?

**Comment by williamium3000** [comment]
Hi, I am also curious about this. Is this no necessary in accelerate or what can I do to set it?

**Comment by sgugger** [comment]
It's not necessary. Accelerate synchronizes the RNGs in your dataloader at the beginning of every epoch automatically.
